OWNERS_ALIASES

```diff
@@ -11,34 +11,25 @@ aliases:
   - strongjz
 
   ingress-nginx-maintainers:
-  - rikatz
-  - strongjz
   - cpanato
+  - Gacko
   - puerco
   - tao12345666333
 
   ingress-nginx-reviewers:
-  - rikatz
-  - strongjz
-  - puerco
   - cpanato
+  - Gacko
+  - puerco
   - tao12345666333
 
   ingress-nginx-helm-maintainers:
-  - cpanato
```

Comment:
why dropping?

Reply:
You're already in the `ingress-nginx-maintainers`, according to docs that's getting inherited.